openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -32,10 +52,10 @@ def __init__(self):
         self.supports['inequality_constraints'] = True
         self.supports['equality_constraints'] = True
         self.supports['multiple_objectives'] = False
-        self.supports['two_sided_constraints'] = True
 
         # TODO: Support these
         self.supports['linear_constraints'] = False
+        self.supports['two_sided_constraints'] = False
```

Comment:
Wait, this is pyoptsparse. Did you merge out? two_sided_constraints should be true in the file and the docstring